build.sbt

```diff
@@ -567,6 +570,7 @@ lazy val scioExamples: Project = Project(
   sources in doc in Compile := List()
 ).dependsOn(
   scioCore,
+  scioBigQuery,
```

Comment:
dependencies on scio-bigquery now have to be explicit since core is not pulling it anymore. We should document that.

Reply:
#1075 might have introduced a conflicting dependency change on `scio-bigquery`. Can you take a look?

Reply:
Should be fixed now